apps/installer/includes/functions.sh

```diff
@@ -27,7 +27,9 @@ function inst_configureOS() {
 
 function inst_updateRepo() {
     cd "$AC_PATH_ROOT"
-    git pull origin $(git rev-parse --abbrev-ref HEAD)
+    source "$AC_PATH_CONF/config.sh.dist" #hack to get the parameter for the branch name
+    echo "Pulling from $INSTALLER_PULL_FROM branch defined in config.sh.dist"
```

Comment:
why from `config.sh.dist` ? shouldn't it pull from `config.sh` ?

@Yehonal

Reply:
Should I add first to check if `config.sh` exist, if not read from `config.sh.dist` ?

Reply:
yes of course ;)

Reply:
what is this...

It's not the way, wait a second

Reply:
As you can see here: https://github.com/azerothcore/azerothcore-wotlk/blob/master/apps/installer/includes/includes.sh

The installer includes the bash_shared/common.sh

https://github.com/azerothcore/azerothcore-wotlk/blob/master/apps/bash_shared/common.sh#L4

Here we already handle the conf.sh and conf.dist.sh for ALL our bash scripts, in fact, you can notice that in other function we directly use configuration variables without including those files.

Can you try without your "source" stuff please?